### ISSUE TITLE
🐛 BUG: Astro v0.21.next wasm panic with missing css file

### ISSUE THREAD

**Issue by andreialecu** [withastro/astro#1800]
### What package manager are you using?

yarn

### What operating system are you using?

Mac

### Describe the Bug

I was testing astro `next` today and got a hard crash because of a leftover in an .astro file:

```html
    <style global lang="scss">
      @use "react-modal-video/scss/modal-video.scss";
    </style>
```

I previously uninstalled that module, but `astro dev` crashed with an error message that was a bit unhelpful:

```
➜ yarn dev
14:34 [astro] Server started                               192ms
14:34 [astro] Local: http://localhost:3000
panic: runtime error: index out of range
wasm://wasm/0025b49a:1


RuntimeError: unreachable
    at <anonymous>:wasm-function[205]:0xdf4c
    at <anonymous>:wasm-function[14]:0x9ce
    at <anonymous>:wasm-function[829]:0x68c42
    at <anonymous>:wasm-function[292]:0x13371
    at <anonymous>:wasm-function[299]:0x14098
    at Go._resume (file:///Users/andreialecu/Work/.../node_modules/@astrojs/compiler/node/wasm_exec.js:374:28)
    at file:///Users/andreialecu/Work/.../node_modules/@astrojs/compiler/node/wasm_exec.js:384:16
```

Related: https://github.com/snowpackjs/astro/pull/1782

### Steps to Reproduce

Add the following in any `.astro` file.
```html
    <style global lang="scss">
      @use "foobar/baz.scss";
    </style>
```

**Comment by natemoo-re** [comment]
Thank you! We'll have to fix this (or at least improve the error message) before `astro@0.21.0` is released.

**Comment by natemoo-re** [comment]
@drwpow Any updates on this one? I feel like you may have fixed it.

**Comment by drwpow** [comment]
Yes I’m almost positive this is https://github.com/withastro/astro/issues/2032 and has been fixed but will confirm.

**Comment by drwpow** [comment]
Just confirmed that this not only shows a helpful error, but the original `@use` code still works if you have the module installed. Fixed as of `0.21.6`.
